<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_edits</class>
 <widget class="QDialog" name="admin_edits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="viewCurrentAccData_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>current username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewCurrentAccData_pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="viewAndEditusername_lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="viewAndEditPassword_lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="viewAndEditAddress_lineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="viewAndEditAge_lineEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editAcc_pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
